dienst2/settings.py

```diff
@@ -83,6 +83,21 @@
     'django.template.loaders.app_directories.Loader',
 )
 
+TEMPLATES = [
```

Comment:
om 1 of andere reden kon ik in de templates niet `request.META` benaderen ofzo, dus toen heb ik dit toegevoegd. 

Als je vraagt waarom het tegenwoordig in een `TEMPLATES` variable zit, dat is omdat je makkelijker je template renderer kan wisselen sinds django 1.8, dus je kan ook per template backend andere settings hebben.